addons/web/static/tests/legacy/views/calendar/calendar_view_tests.js

```diff
@@ -5403,6 +5399,16 @@ QUnit.module("Views", ({ beforeEach }) => {
                 `,
             });
 
+            patchWithCleanup(browser, {
```

Comment:
why not a single patchWithCleanup?

Reply:
would be better to have initial localStorage value different from "month". How do we now your code work if you didn't modified
assert.equal(target.querySelector(".scale_button_selection").textContent, "Month");
?